### PR TITLE
DOT-2154

### PR DESCRIPTION
**Pull request by pinanks**

### CODE REVIEW

File: url.json [hunk 1]
```diff
@@ -1,7 +0,0 @@
-[
-    {
-      "name": "lambdatest-home-page",
-      "url": "https://www.lambdatest.com"
-    }
-  ]
-
```
Comment (sushobhit-lt) [inline url.json:7]: @pinanks `smartui-web.json` and `url.json` file was added purposefully, please add that

File: src/lib/config.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    let filetype = path.extname(filepath);
+    if (filetype != '.json') {
+        console.log('Error: Config file must have .json extension');
+        return
+    }
+
+    // verify the file does not already exist
+    if (fs.existsSync(filepath)) {
+        console.log(`Error: SmartUI Web Config already exists: ${filepath}`);
+        console.log(`To create a new file, please specify the file name like: 'smartui config:create-web webConfig.json'`);
```
Comment (sushobhit-lt) [inline src/lib/config.ts:45]: @pinanks Please add logger instead of console logs

File: src/tasks/auth.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+export default (ctx: Context): ListrTask<Context, ListrRendererFactory, ListrRendererFactory>  =>  {
+    return {
+        title: `Authenticating with SmartUI`,
+        task: async (ctx, task): Promise<void> => {
+            updateLogContext({task: 'auth'});
+
+            try {
+                await ctx.client.auth(ctx.log);
+                task.output = chalk.gray(`using project token '******#${ctx.env.PROJECT_TOKEN.split('#').pop()}'`);
+                task.title = 'Authenticated with SmartUI';
```
Comment (sushobhit-lt) [inline src/tasks/auth.ts:15]: lets use constants for same messages, multiple times like
`'Authenticated with SmartUI'`

File: src/commander/exec.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                        [ListrDefaultRendererLogLevels.OUTPUT]: color.gray
+                    }
+                }
+            }
+        )
+
+        try {
+            await tasks.run(ctx);
+        } catch (error) {
+            console.log('\nRefer docs: https://www.lambdatest.com/support/docs/smart-visual-regression-testing/')
```
Comment (sushobhit-lt) [inline src/commander/exec.ts:52]: let not use  `console.log` also use debug for error

File: src/lib/config.ts (hunk 2 above)
Comment (pinanks) [reply src/lib/config.ts:45]: Will have to use console.log for now (DOT-2260)

**Pull request merged**